doc/swish/db.tex

```diff
@@ -141,16 +141,14 @@
 
 \end{itemize}
 
-\genserver{db}{init} The \code{init} procedure takes a filename and
-mode symbol and attempts to open that database, setting
-\code{journal\_mode} to ``wal'' if \var{mode} is
-\code{create}. The handle returned from \code{osi\_open\_database} is
-wrapped in a \var{database} record that is registered with a
-foreign-handle guardian using the type name \code{databases}.
-The foreign-handle guardian hooks
-the garbage collector so that dead databases are
-closed even if the \code{db} gen-server fails to close them for any
-reason.
+\genserver{db}{init} The \code{init} procedure takes a filename, mode
+symbol, and initialization procedure and attempts to open that
```

Comment:
and **an** initialization procedure

Reply:
I debated that one when I wrote it. Updated.